docs/osquery/osquery.asciidoc

```diff
@@ -51,111 +48,167 @@ and you'll get suggestions for agents by name, ID, platform, and policy.
+
+* To upload queries from a .conf query pack, drag the pack to the drop zone under the query table. To explore the community packs that Osquery publishes, click *Example packs*.
+
+. Click *Save pack*. The queries run when the policy receives the update.
+
+[float]
+[[osquery-schedule-status]]
+=== View status of scheduled packs
+
+. Open the **Packs** tab.
+
+. Click a pack name to view the status.
 +
 Details include the last time each query ran, how many results were returned, and the number of agents the query ran against.
 If there are errors, expand the row to view the details.
 +
 [role="screenshot"]
-image::images/scheduled-query-groupds.png[Shows last results last time it ran, how many results returned, number of agents it ran against, if it is actually running and if there are errors]
+image::images/scheduled-pack.png[Shows queries in the pack and details about each query, including the last time it ran, how many results were returned, the number of agents it ran against, and if there are errors]
 
-. To make changes, click *Edit*.
+. View scheduled query results in <<discover,*Discover*>> or the drag-and-drop <<lens,*Lens*>> editor.
 
-.. To add a query to the group, click *Add query*, and then enter the query ID and interval.
-Optionally, set the minimum Osquery version and platform,
-or <<osquery-map-fields,map ECS fields>>.
+[float]
+[[osquery-manage-query]]
+== Edit saved queries
+
+Add or edit saved queries from the *Saved queries* tab.
+
+. Go to the saved queries, then click **Add saved query** or the edit icon.
+. Provide the following fields:
 
-.. To upload queries from a .conf query pack, drag the pack to the drop zone under the query table. To explore the community packs that Osquery publishes, click *Example packs*.
+* The unique identifier.
 
-. Click *Save query*. The queries run when the policy receives the update.
+* A brief description.
 
-. View scheduled history results in <<discover,*Discover*> or the drag-and-drop <<lens,*Lens*>> editor.
+* The SQL query.
 
+* The <<osquery-map-fields,ECS fields>> to populate when the query is run. These fields are also copied in when you add this query to a pack.
+
+* The defaults to set when you add the query to a pack.
+
+** The frequency to run the query.
+
+** The minimum https://github.com/osquery/osquery/releases)[version of Osquery] required to run the query.
+
+** The operating system required to run the query. For information about supported platforms per table, refer to the https://osquery.io/schema[Osquery schema].
+
+. Click *Test configuration* to test the query and any mapped fields: 
+
+* From the *Test query* panel, select agents or groups to test the query, then click *Submit*. This runs a live query. Result columns that are mapped are marked with image:images/mapped-icon.png[mapping icon]. Hover over the icon to see the mapped ECS field.
```

Comment:
The period after to icon wraps to the next line.  Not sure how to fix other than to remove a word.  You could do "then click *Submit* to run a live query.

Reply:
@gchaps - did the update commit fix what you were seeing? Thanks!